addon/components/dashboard-calendar.js

```diff
@@ -254,24 +264,34 @@ export default Component.extend({
     return types.toArray().sortBy('title');
   }),
 
+  /**
+   * @property vocabularies
+   * @type {Ember.computed}
+   * @public
+   */
+  vocabularies: computed('bestSelectedSchool.vocabularies.[]', async function(){
+    const school = await this.get('bestSelectedSchool');
+    const vocabularies = await school.get('vocabularies');
+    await all(vocabularies.mapBy('terms'));
+    return vocabularies.toArray().sortBy('title');
+  }),
+
   /**
    * @property showClearFilters
    * @type {Ember.computed}
    * @public
    */
-  showClearFilters: computed('selectedCourses.[]', 'selectedSessionTypes.[]', 'selectedCourseLevels.[]', 'selectedCohorts.[]', {
-    get() {
-      const a = this.get('selectedSessionTypes');
-      const b = this.get('selectedCourseLevels');
-      const c = this.get('selectedCohorts');
-      const d = this.get('selectedCourses');
+  activeFilters: computed('selectedCourses.[]', 'selectedSessionTypes.[]', 'selectedCourseLevels.[]', 'selectedCohorts.[]', 'selectedTerms.[]', function () {
+    const a = this.get('selectedSessionTypes');
+    const b = this.get('selectedCourseLevels');
+    const c = this.get('selectedCohorts');
+    const d = this.get('selectedCourses');
+    const e = this.get('selectedTerms');
 
-      let results = a.concat(b, c, d);
-      this.set('activeFilters', results);
+    const activeFilters = [].concat(a, b, c, d, e);
```

Comment:
no need to assign this to a variable 

```
const activeFilters = [].concat(a, b, c, d, e);
return activeFilters;
```

can be shortened to `return [].concat(a, b, c, d, e);`